meinberlin/apps/polls/assets/PollQuestions.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react'
+import PollQuestion from './PollQuestion'
+
+const api = require('adhocracy4').api
+
+export const PollQuestions = (props) => {
+  const [questions, setQuestions] = useState([])
+
+  useEffect(() => {
+    api.poll.get(props.pollId)
```

Comment:
Its telling me props.poll is not defined when saving a question, so I think you also need to adjust the api.poll.change call above?

Reply:
oh thanks, good catch!

You mean the one in `EditPollQuestions.jsx` right? This part (userview) can be buggy, because its very WIP.

Reply:
Oh yes, commented on the wrong file, sorry..